tests/test_utils.py

```diff
@@ -24,3 +25,20 @@ def test_check_paths_exists(test_args):
     # path is not a valid type
     setattr(test_args, "output_data", 123)
     assert check_paths_exists(test_args) == False
+
+def test_read_config_file(good_config):
+
+
+    dict = read_config_file(good_config)
+    assert dict['code'] == 'code'
+    assert dict['input_data'] == 'input_data'
+    assert dict['csv'] == None
```

Comment:
I think the preferred way to check if something is NoneType in python is `assert dict['csv'] is None`